src/lib/ydocker/main_dialog.rb

```diff
@@ -148,22 +156,33 @@ def selected_container
       Docker::Container.get(selected)
     end
 
-    def stop_container
-      return unless (Yast::Popup.YesNo(_("Do you really want to stop the running container?")))
-      selected_container.stop!
-      return unless (Yast::Popup.YesNo(_("Do you want to remove the container?")))
-      selected_container.delete
+    def start_container
+      return unless (Yast::Popup.YesNo(_("Do you really want to start the container?")))
```

Comment:
is this question needed? I think start is non-problematic operation, as you can easily undo it by kiling it ( which is opposite to stop/kill which is harder to undo and can cause operation corruption